clients/fides-js/__tests__/integrations/blueconic.test.ts

```diff
@@ -58,6 +64,22 @@ describe("blueconic", () => {
     ).not.toHaveBeenCalled();
   });
 
+  test("when fides configures no consent, blueconic sets consent for all purposes", () => {
```

Comment:
nit: I'd add an explicit test outside of this case for something like:
```
test("when no approach is specified, 'onetrust' is used by default", ...)
```

...and then I'd wrap these other tests with a describe like:
```
describe("when using the default approach", () => { ... }
```

Right now it's subtle that there are some existing paved roads here to allow creating different approaches in the future.

Reply:
Good call, will do that.

Reply:
I couldn't _quite_ do the first suggestion because there was no direct observable way with this code to check that `'onetrust'` is a "default". So instead I've rewritten the tests to assert that `undefined` and `"onetrust"` approaches being passed in have identical behavior.